Dockerfile.windows

```diff
@@ -0,0 +1,8 @@
+ARG WIN_VERSION
+
+FROM mcr.microsoft.com/windows/nanoserver:${WIN_VERSION}
+
+CMD ["/ryuk.exe"]
+ENTRYPOINT ["/ryuk.exe"]
+
+COPY ryuk.exe /
```

Comment:
```suggestion
ARG WIN_VERSION

FROM mcr.microsoft.com/windows/nanoserver:${WIN_VERSION}

ENTRYPOINT ["/ryuk.exe"]

COPY ryuk.exe /
```

Reply:
Makes sense, let's remove the CMD, as it's not adding anything different from the entrypoint